src/helm/proxy/clients/together_client.py

```diff
@@ -12,6 +12,9 @@
 from .client import Client, wrap_request_time, truncate_sequence
 
 
+MODEL_ALIASES = {"h3-2.7b": "h3-2.7b-h3"}
```

Comment:
Could you write a comment to provide the context (that we discussed)?

Reply:
Added docstring.